tools/autograd/derivatives.yaml

```diff
@@ -2799,8 +2799,14 @@
   lengths: non_differentiable
   dummy: non_differentiable
 
+- name: _nested_view_from_jagged_with_cache(Tensor(a) self, Tensor offsets, Tensor dummy, Tensor? lengths=None, int ragged_idx=1, int min_seqlen=-1, int max_seqlen=-1) -> Tensor(a)
+  self: grad.values()
+  offsets: non_differentiable
+  lengths: non_differentiable
+  dummy: non_differentiable
+
 - name: _nested_get_values(Tensor(a) self) -> Tensor(a)
-  self: _nested_view_from_jagged(grad, at::_nested_get_offsets(self), at::_nested_get_jagged_dummy(self), at::_nested_get_lengths(self), at::_nested_get_ragged_idx(self))
+  self: _nested_view_from_jagged_with_cache(grad, at::_nested_get_offsets(self), at::_nested_get_jagged_dummy(self), at::_nested_get_lengths(self), at::_nested_get_ragged_idx(self), at::_nested_get_min_seqlen(self), at::_nested_get_max_seqlen(self))
```

Comment:
Would be nice to have a test that repro's @YuqingJ's original error. Maybe compile + using this function + backward would do the trick?

Reply:
yep agreed, i did add a test that fails propagating the cache, but we could go a step further and trigger the new error from Brian's recent PR. I'll look into it

Reply:
Okay looks like adding a `backward()` call does the trick; I was able to trigger the error